docs/en/guides/How-to-build.md

```diff
@@ -17,11 +17,19 @@ If you need to execute build behind the proxy, edit the *.mvn/jvm.config* and pu
 
 ### Build from GitHub
 1. Prepare git, JDK8 and maven3
-1. `git clone https://github.com/apache/skywalking.git`
-1. `cd skywalking/`
+1. Clone project
+    ```bash
+    git clone --recurse-submodules https://github.com/apache/skywalking.git
+    cd skywalking/
+    
+    OR
+    
+    git clone https://github.com/apache/skywalking.git
+    cd skywalking/
+    git submodule init
+    git submodule update
```

Comment:
Your step is not right, because branch change could relate to submodule change.

Reply:
Got it.

Reply:
Maybe we could specify the branch name / tag name in the after `git clone command`.